kubernetes/icinga2/files/scripts/election.py

```
@@ -0,0 +1,236 @@
+    #    (kubernetes leader election library exits after OnStoppedLeading callback)
+    log.info("Demote complete, exiting election client")
+    sys.exit(0)
+
+
+def trigger_reload():
+    """POST /v1/actions/restart-process on local Icinga2."""
+    import requests
+    try:
+        resp = requests.post(
+            f"{API_URL}/v1/actions/restart-process",
+            auth=(API_USER, API_PASS), verify=False,
```

Ruff (0.16.0)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)